### ISSUE TITLE
headlag

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5554]
**Preferred term label:**
Head lag

**Synonyms**
Headlag

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4690596/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035241/ OMIM:618744 
frequency 18/18, infantile onset

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Also investigate term for ability to roll over, 18/18 infants could not in the above publication

**Comment by LCCarmody** [comment]
It is in this table. Of the 18 patients with homozygous variants, they all had headlag.

[Supplementary Table 1: Extended clinical characteristics of 18 patients with homozygous UGP2 variants (XLSX 32 kb)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035241/bin/401_2019_2109_MOESM4_ESM.xlsx)
